test/resources/tasks.test.ts

```diff
@@ -0,0 +1,65 @@
+import { Applicant, Task } from "onfido-node";
+
+import { exampleTask } from "../test-examples";
+import {
+  cleanUpApplicants,
+  cleanUpWebhooks,
+  createApplicant,
+  completeTask,
+  createWorkflowRun,
+  getExpectedObject,
+  onfido
+} from "../test-helpers";
+
+function getExpectedTask(exampleTask: Task, overrideProperties = {}) {
+  return getExpectedObject(exampleTask, {
+    task_def_id: expect.stringMatching(/^[0-9a-z-_]+$/),
+    task_def_version: null,
+    workflow_run_id: expect.stringMatching(/^[0-9a-z-]+$/),
+    created_at: expect.anything(),
+    updated_at: expect.anything(),
+    ...overrideProperties
+  });
+}
+
+const workflow_id = "e8c921eb-0495-44fe-b655-bcdcaffdafe5";
+let applicant: Applicant;
+let workflowRunId: string;
+
+beforeEach(async () => {
+  applicant = (await createApplicant()).data;
+  workflowRunId = (await createWorkflowRun(applicant, workflow_id)).data.id;
+});
+
+afterAll(() => {
+  return Promise.all([cleanUpApplicants(), cleanUpWebhooks()]);
```

Comment:
I believe `cleanUpWebhooks()` is useless here and I guess we don't need a `cleanUpWorkflows()` since they should go away with the applicants' deletion.

Reply:
Good point, removed.